docs/APISPEC.md

```diff
@@ -2,221 +2,1225 @@
 
 ## General info
 
-- Ratelimit: 10 req/m
+Testaustime API gives 5 different routes:
+- [/auth/](#auth)
+- [/users/](#users)
+- [/activity/](#activity)
+- [/friends/](#friends)
+- [/leaderboards/](#leaderboards)
 
-The desired interval at which to send heartbeats is immediately when editing a file, and after that at max every 30 seconds, and only when the user does something actively in the editor.
+Basic path: `https://testaustime.fi/api`
```

Comment:
The path should be https://api.testaustime.fi/ instead.
Same path should be used throughout the documentation.